src/osd/ReplicatedBackend.cc

```diff
@@ -154,6 +157,19 @@ bool ReplicatedBackend::can_handle_while_inactive(OpRequestRef op)
   }
 }
 
+bool ReplicatedBackend::can_handle_fast(OpRequestRef op)
```

Comment:
Ok, you can't call this can_handle_fast.  Other parts of the code base use that to refer to the messenger fast_dispatch machinery.